packages/core/src/expression-builders/identifier.ts

```diff
@@ -16,7 +17,9 @@ export class Identifier extends BaseSqlExpression {
  * Unlike {@link attribute} and {@link col}, this identifier will be escaped as-is,
  * without mapping to a column name or any other transformation.
  *
- * @param value
+ * This method supports strings, table structures, model classes (which), and model definitions
```

Comment:
What do you mean with `(which)` here?

Reply:
That's part of a comment I deleted 😅
I'll remove it

Reply:
Done!